Objects/object.c

```diff
@@ -2206,6 +2206,15 @@ _Py_Dealloc(PyObject *op)
     (*dealloc)(op);
 }
 
+
+PyObject **
```

Comment:
Can you mark this function as `inline` at least?

Reply:
I don't know what is the effect of adding "inline" here. My intent is that when a C extension uses PyObject_GET_WEAKREFS_LISTPTR(), it calls this function.

If a C extension inlines PyObject_GET_WEAKREFS_LISTPTR(), this change is basically useless.

If you consider that PyObject_GET_WEAKREFS_LISTPTR() must continue to be "inlined" in CPython code base, I add a private static inline function to the **internal** C API, and export PyObject_GET_WEAKREFS_LISTPTR() as a regular function in the **public** C AP.

Reply:
I think we should still have a private inlined version, but otherwise LGTM

Reply:
> I think we should still have a private inlined version

Done, but only added to the internal C API.